### PR TITLE
Celo crowdfunding tutorials

### PR DESCRIPTION
**Pull request by alexreyes**
Hi, 

I am submitting these three Celo learn tutorials for the hackathon on behalf of me and [@neocho](https://github.com/neocho). 

We created three new Celo tutorials, all based on our [hackathon project](https://github.com/Alex-Neo-Projects/Coperacha-app) and the [smart contracts](https://github.com/Alex-Neo-Projects/Coperacha-contracts) we wrote for the hackathon. 

The tutorials are all around creating a crowdfunding smart contract. There are three parts:

[1. Building a Crowdfunding Smart Contract in Celo](https://github.com/alexreyes/Celo-Crowdfunding-Tutorial)
[2. Deploying a Crowdfunding Smart Contract in Celo](https://github.com/alexreyes/Celo-Crowdfunding-Tutorial-2)
[3. Interacting with the Crowdfunding Smart Contracts](https://github.com/alexreyes/Celo-Crowdfunding-Tutorial-3)

All of which are in this PR. Let us know all the feedback you have!

### CODE REVIEW

**Review by pyr0gan — COMMENTED** [review]
I have only minor concerns I'd like to address. I feel it would be best to add a consistent link to the Learn Discord where you mention it at the bottom of each of the tutorials. Not an immediate need, but for the sake of completeness, right?

When you talk about public variables, it might be beneficial to cover the circumstances in which a public variable is useful vs. when it is not. Perhaps provide a link to further detail, such as : https://www.bitdegree.org/learn/solidity-variables#control-variable-visibility 

I enjoy this triptych of tutorials, though there are a small number of inconsistencies. Sometimes when you reference a function name, you have not included the parentheses indicating it is a function. This is far from a critical issue, but I feel it bears mention. "Let's" reads better as "Let us". Try to keep "you" and "we" clear and consistent throughout the tutorials - I see you have done a good job of this, with only a few standouts. Easy polish!

As I aim to be a very security conscious developer, I tend to want to break out an explanation of using things like .gitignore to shield the contents of a .env file from being checked out to a git repository any time .env is even mentioned in passing. We don't want developers reading this content to get the notion that they don't need to be responsible for the safety of their API keys - especially DataHub API keys!
Please consider including a sentence or two about the importance of keeping API keys and the contents of an .env file safe!

@alexreyes - minor consistency fixes and links to the Discord please. This is well written and clear, otherwise! 👍🏻

**Comment by alexreyes** [comment]
@pyr0gan Thanks for the detailed feedback. I went in and made the changes you suggested, and updated the tutorials. Let me know what you think.

**Review by pyr0gan — COMMENTED** [review]
Looking good, just a couple of additional touch-ups needed.

File: network-documentation/celo/tutorial/celo-crowdfunding/3.interacting.md [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
+The first step is to create a Javascript file to write our smart contract interactions in. In the root of the ``celo-crowdfunding`` folder, create a file named ``interact.js``.  
 
 We will use the packages we installed via NPM in the first tutorial here. We will also need one additional module, BigNumber, in order to work with the large numbers the Celo blockchain uses. 
 
-In your terminal, run: 
+In the terminal, run: 
 
 ``npm install bignumber.js``
 
 And that's it for our setup!
```
[the remark this answers is not in the document]
Comment (alexreyes) [reply network-documentation/celo/tutorial/celo-crowdfunding/3.interacting.md:37]: Good point, I added this to the latest commit 👍

File: network-documentation/celo/tutorial/celo-crowdfunding/3.interacting.md [hunk 2]
```diff
@@ -79,7 +91,7 @@ async function interact()  {
 interact();
```
[the remark this answers is not in the document]
Comment (alexreyes) [reply network-documentation/celo/tutorial/celo-crowdfunding/3.interacting.md:93]: I updated the latest commit to mention that the .env file is created in part two, and I changed the variable type to not be a constant 👍

**Comment by alexreyes** [comment]
@pyr0gan Just updated the PR with the comments you made

**Comment by pyr0gan** [comment]
> @pyr0gan Just updated the PR with the comments you made

I'll coordinate with @yash-sharma1 to merge and announce this tonight 👍🏻

**Pull request merged**